src/main/java/ro/siit/trips/controllers/TripController.java

```diff
@@ -0,0 +1,190 @@
+package ro.siit.trips.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.ModelAndView;
+import ro.siit.trips.models.Person;
+import ro.siit.trips.models.Trip;
+import ro.siit.trips.repositiories.PersonRepository;
+import ro.siit.trips.repositiories.TripRepository;
+import ro.siit.trips.services.FileService;
+import ro.siit.trips.services.TripService;
+
+import javax.validation.Valid;
+import java.util.UUID;
+
+
+@Controller
+public class TripController {
+
+    @Autowired
+    private TripRepository tripRepository;
+    @Autowired
+    private PersonRepository personRepository;
+    @Autowired
+    private FileService fileService;
+
+    @Autowired
+    private TripService tripService;

```

Comment:
If you use a TripService then you should clearly separate your concerns so: Controller should only call Service which should call Repository.
Controller should not call Repository directly.